CHANGELOG.md

```diff
@@ -9,19 +9,28 @@ was committed to the `master` branch. This is not necessarily the date that
 the code was deployed.
 
 ## [Unreleased]
+### Added
```

Comment:
Haha, did we put all these changelog lines under the wrong release. Oops. Thanks for fixing it. I think the linting one should also be in `[Unreleased]`.